src/components/common/applicant-info-1.vue

```diff
@@ -329,10 +329,15 @@
 
 <script lang="ts">
 import { Component, Vue, Watch } from 'vue-property-decorator'
-import newReqModule from '@/store/new-request-module'
+import { Action, Getter } from 'vuex-class'
+
 import ApplicantInfoNav from '@/components/common/applicant-info-nav.vue'
 import { Location } from '@/enums'
 import { NameRequestMixin } from '@/mixins'
+import { ActionBindingIF } from '@/interfaces/store-interfaces'
+import { ApplicantI, NameRequestI, SubmissionTypeT } from '@/interfaces'
+import $canJurisdictions from '@/store/list-data/canada-jurisdictions'
+import $intJurisdictions from '@/store/list-data/intl-jurisdictions'
```

Comment:
What do you think about these -- should be part of the store (instead of just in the store folder)? Or perhaps a separate resource folder?

BTW I don't like their name. `$xxx` is used in other frameworks to identify an [observable](https://www.freecodecamp.org/news/what-are-observables-how-they-are-different-from-promises/). In Vue perhaps they refer to Vue's internal methods (eg, "$refs"). Naming them as they are here is misleading and giving them too much credit.

Reply:
So I've decided that I'll continue to attach these "code tables" (static data) to the Vue instance, so that they don't get imported multiple times (in different components).
I'm still working on cleaning this up...